src/assignments/orbit-report.rst

```diff
@@ -1,282 +1,351 @@
-
 Assignment #6: Orbit Report
 ===========================
 
-.. warning::
+.. admonition:: Warning
 
-   Please do not attempt this assignment until AFTER your first lecture on Angular.
-   It can be tempting to dive right in, but Angular is a broad topic and you want to wait to start until you have time to go over the requirements in class.
+   Please do not attempt this assignment until AFTER your first lesson on
+   Angular. It can be tempting to dive right in, but Angular is a broad topic,
+   and you want to wait to start until you have time to go over the
+   requirements in class.
 
-There are thousands of satellites orbiting the earth. You are tasked with creating a searchable, sortable table of satellites.
-For the purposes of this assignment, **satellite** will be defined as any object purposefully placed into orbit.
+There are thousands of satellites orbiting the earth. You are tasked with
+creating a searchable, sortable table of satellites. For the purposes of this
+assignment, a **satellite** will be defined as any object purposefully placed
+into orbit.
 
-.. TODO: screen shot of finished report in the browser with numbers by each feature
+Your table will have the following features:
 
-1. **Satellites**: Table with each row being a satellite.
-2. **Counts:** Count of all satellites and count for each type of satellite.
-3. **Search form:** Text entered limits results based on matches. Pressing enter or clicking button triggers search.
-4. **Sortable columns:** The name and type column headers can be clicked, which will sort the table using that property.
+#. **Satellites**: Each row in the table contains data on one satellite.
+#. **Search form:** Filters the results based on matches to the entered text.
+   Pressing enter or clicking the button triggers the search.
+#. **Sortable columns:** The ``Name`` and ``Type`` column headers can be
+   clicked, which will sort the table using that property.
+#. **Counts:** Displays the total number of satellites in the table as well as
+   the count for each type of satellite.
 
+Your completed assignment should look something like this:
 
-Setup
------
+.. figure:: figures/orbit-report-table.png
+   :alt: Screenshot of orbit report table.
+
+You can also view a video of the finished project in the
+:ref:`Orbit Report Demo <orbit-report-demo>` section at the bottom of this
+page.
 
-1. Create a new project on Github named ``orbit-report``.
-2. Clone that repo to your computer.
-3. Create a new Angular project in the cloned repo folder using this command.
+Setup
```

Comment:
Your method was the one I used when I tried the assignment myself, but I did not want to overwrite Blake's instructions.  I prefer our way (especially since it does not double-name the project), so I re-worked the steps.

Let me know what you think.

Reply:
yeah looks good